### PR TITLE
Replace deprecated Page.Dir with Page.File.Dir

### LINKED ISSUE #986: Build fails with hugo 0.92.0+
> Error: Error building site: "/home/jplatte/code/matrix-spec/content/client-server-api/_[index.md:2293](http://index.md:2293/):1": failed to render shortcode "cs-module": failed to process shortcode: "/home/jplatte/code/matrix-spec/layouts/shortcodes/cs-module.html:12:42": execute of template failed: template: shortcodes/cs-module.html:12:42: executing "shortcodes/cs-module.html" at <.Page.Dir>: can't evaluate field Dir in type [page.Page](http://page.page/)[](https://matrix.to/#/!NasysSDfxKxZBzJJoE:matrix.org/$ykRSehREShnNAHpuVfx0NRi9oaW9gQmOMhFB5VE8uWA?via=matrix.org&via=libera.chat&via=element.io)

### PR DESCRIPTION
**Pull request by anoadragon453**
Fixes #986 (which has a bit more context).

`Page.Dir` has been deprecated for a while, calling them raised an error in [hugo v0.92.0](https://github.com/gohugoio/hugo/releases/tag/v0.92.0), before it was finally removed in [hugo v0.93.0](https://github.com/gohugoio/hugo/releases/tag/v0.93.0).

This commit replaces instances of `Page.Dir` with [`Page.File.Dir`](https://gohugo.io/variables/files/).

Preview: https://pr988--matrix-spec-previews.netlify.app

### CODE REVIEW

File: layouts/shortcodes/cs-module.html [hunk 1]
```diff
@@ -9,5 +9,5 @@
 
 {{ $name := .Params.name }}
 
-{{ $page := .Site.GetPage (path.Join .Page.Dir "modules" (printf "%s%s" $name ".md"))}}
-{{ $page.Content }}
\ No newline at end of file
+{{ $page := .Site.GetPage (path.Join .Page.File.Dir "modules" (printf "%s%s" $name ".md"))}}
```
Comment (anoadragon453) [inline layouts/shortcodes/cs-module.html:12]: I'm not 100% sure where the changelog for these changes should go...

Comment (turt2live) [reply layouts/shortcodes/cs-module.html:12]: we don't have a changelog place for this PR's changeset.

**Review by turt2live — COMMENTED** [review]
the changes look good, and it appears as though Hugo fixed whatever was causing this to not work on Windows. The deprecated version was previously chosen for compatibility reasons.

Can we also bump the minimum required version in the README (currently v0.74) to prevent future issues? Arbitrarily setting it to v0.93 would be fine, I think.

**Comment by anoadragon453** [comment]
@turt2live bumped min hugo to v0.93.0 in the README and switched our CI to the latest release (v0.93.3).

**Comment by turt2live** [comment]
(please don't rebase after review has been given - it makes things difficult. Merge commits will be squashed out of existence when the PR is merged)

**Comment by anoadragon453** [comment]
(Apologies! I had rewritten a commit that hadn't yet been reviewed, hence assuming it was fine.)

**Review by turt2live — APPROVED** [review]
thanks :)

**Pull request merged**